src/main/java/at/hannibal2/skyhanni/features/misc/halloweenlobbywaypoints/Basket.kt

```diff
@@ -0,0 +1,34 @@
+package at.hannibal2.skyhanni.features.misc.halloweenlobbywaypoints
+
+import at.hannibal2.skyhanni.utils.LorenzVec
+
+enum class Basket(
```

Comment:
Generally, I would prefer the repo instead of enums like this. You could define the basketName with a function instead of using a static variable instead.
Since it's a one-time event, we can keep it outside repo if you prefer that.